### DISCUSSION TITLE
Change the Language (to Portuguese) pt-br

### DISCUSSION THREAD

**Discussion by GabrielCoderz** [perspective-dev/perspective#2450] [General]
### How can i change the language ?

i've trying to change the elements in the css, but i think it's not effective. 

I found some difficulties selecting some elements to change the content, mainly the '::before' pseudo-element.

Can you help me define a way to solve this problem?

**Comment by texodus** [comment]
Internationalization of the interface is not currently supported.

**Reply by texodus** [reply]
Internationalization support has been added in #2565
